packages/fields/src/Controller.js

```diff
@@ -8,6 +8,7 @@ export default class FieldController {
     this.type = config.type;
     this.maybeAccess = config.access;
     this.isPrimaryKey = config.isPrimaryKey;
+    this.isReadOnly = config.isReadOnly;
```

Comment:
Move this onto adminConfig: {
  isReadOnly: true
}

Reply:
this is now works as adminConfig only.